README.md

```diff
@@ -0,0 +1,270 @@
+```
+### Install
+
+Install this project with:
+
+```
+  npm init -y && npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  cd /web-development-course-capstone
+  code .
+```
+Then install live server and open the `html.index` with the live server.
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+```sh
+ npx stylelint "**/*.{css,scss}"
+ npx hint .
+ npx eslint .
+```
+
+ ### Deployment
+
+You can deploy this project using:
+
+```sh
+ N/A
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<!-- > This project is authored by -->
+
+👤 **Zewdie Habtie**
+
+- GitHub: [ZewdieMc](https://github.com/ZewdieMc)
+- Twitter: [@HabtieZewdie](https://twitter.com/HabtieZewdie)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/zewdie-habtie-sisay-947153172)
+
+👤 **Yosef Geda**
+
+- GitHub: [Yosef](https://github.com/Yosefgeda)
+- Twitter: [Yosef](https://twitter.com/AmaneulBe)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ameneul-b-5545)
+
+👤 **Shubham Bhoyar**
+
+- GitHub: [Shubham](https://github.com/shubhambhoyar077)
+- Twitter: [Shubham](https://twitter.com/AmaneulBe)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ameneul-b-5545)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+
+- **Rebuilt the interfafce using react components**
+- **Test the application using Jest**
+- **Add Backend endpoint using NodeJs**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+<!-- > Write a message to encourage readers to support your project -->
+
+If you like this project give it a start on GitHub
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank [Microverse](https://www.microverse.org/) for providing this design and [Cindy Shin in Behance](https://www.behance.net/adagio07), the designer!
+- I would like to thank my coding partners [Shubham](https://github.com/shubhambhoyar077) and [Yosef](https://github.com/Yosefgeda/).
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- FAQ (optional) -->
+
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] [OPTIONAL] Kindly this link is not working, it would be nice if you create your own **MIT** `Licence` 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/217884353-8901430b-78f0-452e-a214-71044f5f6633.png)

Reply:
Fixed, please [have a look](https://github.com/ZewdieMc/web-development-course-capstone/pull/1/commits/c7b0e8d2cadedc0177471f071a044a1dbaae5c58#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R114)

Reply:
So Nice my friend